### ISSUE TITLE
Request to provide torchscript-ed model

### ISSUE THREAD

**Issue by csukuangfj** [facebookresearch/fairseq#4853]
## 🚀 Feature Request

https://huggingface.co/facebook/wav2vec2-base-960h/tree/main
provides only `model.state_dict()`.  Please also consider upload a torchscript-ed model using either `torch.jit.script()`
or `torch.jit.trace()`.

### Motivation
It simplifies users' life as they don't need to install fairseq to use a torchscript-ed model

### Additional context

We want to test models trained using CTC loss from various frameworks using [k2](http://github.com/k2-fsa/k2) for decoding.
(See https://github.com/k2-fsa/k2/pull/1096)

We only need a torchscript-ed model but for the current situation, we have to install fairseq and export the pretrained model by ourselves via torchscript.
